tests/dummy/app/templates/application.hbs

```diff
@@ -1,10 +1,10 @@
-<h2 id='title'>ember-undo-stack</h2>
+<h2 id="title">ember-undo-stack</h2>
```

Comment:
Fixes template linting errors. Use `"` instead of `'`

Reply:
was this manual or is there a codemod? IIRC @pizza 	tried to do this once in the embercom codebase with regexes and got into rebase hell